src/mcu.cpp

```diff
@@ -216,7 +216,14 @@ void MCU_DeviceWrite(mcu_t& mcu, uint32_t address, uint8_t data)
     case DEV_IPRD:
         break;
     case DEV_PWM1_DTR:
-        LCD_SetContrast(*mcu.lcd, 16 - (data >> 4));
+        if (*mcu.is_jv880)
```

Comment:
`*` is used when dereferencing a pointer element.
However `is_jv880` is `int` not a pointer so `*` is not needed.
Use `mcu.is_jv880`